dsconfig/dump.py

```diff
@@ -37,6 +37,31 @@ def get_db_data(db, patterns=None, **options):
             servers = get_servers_with_filters(dbproxy, **kwargs)
             data.servers.update(servers)
 
+    # dump class properties upon request
+    if options.get('class_properties', False):
+        # go through all classes in servers
+        for server, instances in data.servers.to_dict().items():
+            for instance, classes in instances.items():
+                for cls, devices in classes.items():
+                    # skip classes already dump
+                    if cls not in data.classes.keys():
+                        # class properties
+                        for prop, value in db.get_class_property(cls, list(db.get_class_property_list(cls))).items():
+                            if value:
+                                value = [str(v) for v in value]
+                                data.classes[cls].properties[prop] = value
+
+                        # attribute properties
+                        if not options.get('attribute_properties', False):
```

Comment:
`attribute_properties` can not be set from the cli. (in the cli, there is an option named `--no-attribute-properties`)